hive/src/itest/java/org/opensearch/hadoop/integration/hive/AbstractHiveReadJsonTest.java

```diff
@@ -150,30 +150,13 @@ public void testMissingIndex() throws Exception {
         assertEquals(0, result.size());
     }
 
-    @Test
-    public void testParentChild() throws Exception {
-        OpenSearchAssume.versionOnOrBefore(OpenSearchMajorVersion.V_5_X, "Parent Child Disabled in 6.0");
-        String create = "CREATE EXTERNAL TABLE jsonchildread" + testInstance + " (data STRING) "
-                + tableProps("json-hive-pc/child", "'es.index.read.missing.as.empty' = 'true'", "'es.output.json' = 'true'");
-
-        String select = "SELECT * FROM jsonchildread" + testInstance;
-
-        System.out.println(HiveSuite.server.execute(create));
-        List<String> result = HiveSuite.server.execute(select);
-        assertTrue("Hive returned null", containsNoNull(result));
-        assertTrue(result.size() > 1);
-        assertContains(result, "Marilyn");
-        assertContains(result, "last.fm/music/MALICE");
-        assertContains(result, "last.fm/serve/252/2181591.jpg");
-    }
-
     @Test
     public void testNoSourceFilterCollisions() throws Exception {
 
         String create = "CREATE EXTERNAL TABLE jsonartistscollisionread" + testInstance + " (data INT, garbage INT, garbage2 STRING) "
                 + tableProps(
                     resource("json-hive-artists", "data", targetVersion),
-                    "'es.output.json' = 'true'",
+                    "'opensearch.output.json' = 'true'",
                     "'es.read.source.filter'='name'"
```

Comment:
```suggestion
                    "'opensearch.read.source.filter'='name'"
```

Reply:
all of these will be handled in separate PRs